addons/pos_restaurant/models/pos_order.py

```diff
@@ -162,22 +162,33 @@ def _get_fields_for_draft_order(self):
                     ])
         return fields
 
+    def _get_domain_for_draft_orders(self, id_to_search):
+        """ Get the domain to search for draft orders depending on if the related pos.config has
+        an associated floorplan or not.
+
+        :param id_to_search: Id of a table or order.
+        :type id_to_search: int.
+        "returns: list -- list of tuples that represents a domain.
+        """
+        if self._context.get("search_for_tables"):
```

Comment:
return directly `[("state", "=", "draft"), ("id", "=", id_to_search)]` with no `if` statement. The logic in this module must remain the same.